packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -109,473 +104,589 @@ const PersonalDetailsForm = ({
+                                        className='details-form__tin-confirm'
+                                        data-lpignore
+                                        onChange={() =>
+                                            setFieldValue(
+                                                'tax_identification_confirm',
+                                                !values.tax_identification_confirm,
+                                                true
+                                            )
+                                        }
+                                        value={values.tax_identification_confirm}
+                                        label={localize(
+                                            'I hereby confirm that the tax information I provided is true and complete. I will also inform {{legal_entity_name}} about any changes to this information.',
+                                            {
+                                                legal_entity_name: getLegalEntityName('maltainvest'),
+                                            }
+                                        )}
+                                        renderlabel={title => (
+                                            <Text size='xs' line_height='s'>
+                                                {title}
+                                            </Text>
+                                        )}
+                                        withTabIndex={0}
+                                        data-testid='tax_identification_confirm'
+                                    />
+                                )}
+                            </React.Fragment>
+                        )}
+                        {!is_svg && 'account_opening_reason' in values && (
+                            <AccountOpeningReasonField
+                                required
+                                account_opening_reason_list={account_opening_reason_list}
+                                setFieldValue={setFieldValue}
+                                editable_fields={editable_fields}
+                            />
+                        )}
+                    </fieldset>
+                </FormBodySection>
+            </div>
+
+            {is_svg && (
+                <div className='account-form__poi-additional-information'>
+                    <FormSubHeader title={localize('Additional information')} />
+                    {'phone' in values && (
+                        <PhoneField
+                            value={values.phone}
+                            editable_fields={editable_fields}
+                            has_real_account={has_real_account}
+                            required
+                        />
                     )}
-                </fieldset>
-            </FormBodySection>
-        </div>
+                    {!is_high_risk && is_svg && (
```

Comment:
is_high_risk in client store is based on landing companies and is not for MT5 restricted countries. As per the card requirement, we need to show this additional KYC,  where we offer BVI, Vanuatu and Labuan. Can you please double check this and check with BE?

Reply:
@amina-deriv this flow is there in the figma link attached in the card and I've confirmed with Soban that we won't show these additional fields to high_risk clients

Reply:
based on the discussion, this condition for high risk clients has be removed